code/obj/machinery/computer/atmosphere.dm

```diff
@@ -112,7 +112,12 @@ Atmos alert computer
 /obj/machinery/computer/atmosphere/alerts/proc/triggerAlarm(var/class, area/A, var/O, var/alarmsource)
 	if(status & (BROKEN|NOPOWER))
 		return
-	var/list/L = src.alarms[class]
+
+	var/indexclass = class
+	if(class in list("Fire", "Low Pressure", "Flammable Atmosphere", "Flood", "Manual Trip"))
+		indexclass = "Fire"
+	var/list/L = src.alarms[indexclass]
+
```

Comment:
Always make the alarm type Fire?

Reply:
Resolving this as the code meaning is a bit different in the new commit